ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -260,6 +268,7 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
         verifiableProperties.getIntInRange(storeReadWriteEnableSizeThresholdPercentageDeltaName, 5, 0,
             storeReadOnlyEnableSizeThresholdPercentage);
     storeValidateAuthorization = verifiableProperties.getBoolean("store.validate.authorization", false);
+    storeEnableDataPreFetch = verifiableProperties.getBoolean("store.enable.data.pre.fetch", true);
```

Comment:
false by default since that is the current behavior

Reply:
fixed.